docs/deepnote-mcp.md

```
@@ -0,0 +1,189 @@
+https://deepnote.com/mcp
+```
+
+## Authentication
+
+Every request to the MCP server must be authenticated. There are two supported methods.
+
+### API key
+
+Create an API key in your workspace under **Settings & members > Security > API keys**, then configure your MCP client to send it as a bearer token:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)